GenderPayGap.WebUI/appsettings.LOCAL.json

```diff
@@ -2,5 +2,6 @@
   "DaysToKeepBackupFiles": 0,
   "GoogleAnalyticsAccountId": "UA-93591336-1",
   "FeatureFlagNewReportingJourney": "true",
-  "FeatureFlagNewManageOrganisationsJourney": "true"
+  "FeatureFlagNewManageOrganisationsJourney": "true",
+  "ReportingStartYearsToExcludeFromLateFlagEnforcement": [2019]

```

Comment:
Did this use to be [2019,2020]? Where was this set?

Reply:
No, it wasn't set locally; it defaults to [].
On the other environments this is set during the release

Reply:
I didn't notice this file was for local development. That makes sense, thanks 😄.